packages/context/README.md

```diff
@@ -55,6 +52,7 @@ const propTypes = {
   notAuthenticated: PropTypes.node, // react component displayed during authentication
   notAuthorized: PropTypes.node, // react component displayed in case user is not Authorised
   authenticating: PropTypes.node, // react component displayed when about to redirect user to be authenticated
+  callbackComponentOverride: PropTypes.node, // react component displayed when user is connected
```

Comment:
I think we should be uniform with the override components names. Your name is pretty good, but if we change the others, we create breaking changes. @guillaume-chervet What do you think ?

Reply:
You right but as "callback" props is already used I thought it would be a little bit confused to use the same props name. But tell me what you want and I change it ;)

Reply:
Sure, I have few breaking change in mind for 4.0.0.
Let's open some issues about it.